package/lib/compileFunctions.js

```diff
@@ -55,6 +55,10 @@ module.exports = {
          || _.get(this, 'serverless.service.provider.vpc') });
       }
 
+      if (funcObject.maxInstances) {
+        _.assign(funcTemplate.properties, { maxInstances: _.get(funcObject, 'maxInstances')})
```

Comment:
Let's resort to lodash only if there's no native counterparts. Above can be simply written as:

```javascript
funcTemplate.properties.maxInstances = funcObject.maxInstances;
```
```

Reply:
@medikoo updated the code, ran the linter and tests, and CI now builds.  Thanks for looking at it for me.